src/adapters/suggest.js

```diff
@@ -62,11 +62,33 @@ export default class Suggest {
           suggestItems = suggestItems.concat(favorites.slice(0, nbDisplayedFavorites));
         }
 
-        return renderStaticReact(
-          <Fragment>
-            {suggestItems.map((item, index) => <SuggestItem item={item} key={index} />)}
-          </Fragment>
+        const existingElem = document.getElementById('react-suggests-' + tagSelector);
+        if (existingElem) {
+          ReactDOM.unmountComponentAtNode(existingElem);
+          existingElem.remove();
+        }
+
+        const elem = document.createElement('div');
+        elem.setAttribute('id', 'react-suggests-' + tagSelector);
+        this.searchInputDomHandler.parentNode.insertBefore(
+          elem,
+          this.searchInputDomHandler.nextSibling
         );
+
+        const typedValue = this.searchInputDomHandler.value;
+
+        ReactDOM.render(
```

Comment:
This will be mounted each time \renderItems` is called. I guess performances should be the same as static rendering this way.